10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>MD integration is also not new to users - you can probably merge it with the section above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>